src/types.js

```diff
@@ -95,6 +95,20 @@ export type Account = {|
    */
   zulipVersion: ZulipVersion | null,
 
+  /**
+   * An integer indicating what features are available on the server.
+   *
+   * The feature level increases monotonically; a value of N means the
+   * server supports all API features introduced before feature level N.
+   * This is designed to provide a simple way for mobile apps to decide
+   * whether the server supports a given feature or API change.
+   *
+   * Like zulipVersion, we learn the feature level from /server_settings
+   * at the start of the login process, and again from /register when
+   * setting up an event queue.
```

Comment:
This is what we should do, but we don't in this version :wink: 

Look at where `zulipVersion` gets set in `accountsReducer.js`. One of those is missing the corresponding logic for `zulipFeatureLevel`.

Reply:
This is still missing. Without it, this PR really doesn't work properly at all.

Search for `zulipVersion` in `accountsReducer.js`. Notice the mention in the `realmInit` function. That's how we implement this part:
> and again from /register when setting up an event queue.

We need to do the same for `zulipFeatureLevel`.